test/gtest/uct/test_peer_failure.cc

```diff
@@ -300,6 +300,10 @@ UCS_TEST_SKIP_COND_P(test_uct_peer_failure, purge_failed_peer,
             UCS_TEST_ABORT("AM Short failed with " << ucs_status_string(status));
         }
 
+        /* disable UD async progress in order to prevent returning UCS_ERR_BUSY
+         * from uct_ep_pending_add() below */
```

Comment:
why not just handle BUSY return code?
e.g first pending_add can have BUSY, but once we get OK - the rest should also be OK

Reply:
> why not just handle BUSY return code?

@yosefe do you mean check for BUSY? and if it returns BUSY, skip the test?

Reply:
we can either push more AMs, or pass the test (e.g not fail if returned BUSY)

Reply:
done